queries/java/folds.scm

```diff
@@ -4,4 +4,5 @@
   (constructor_declaration)
   (argument_list)
   (annotation_argument_list)
+  (import_declaration)+
```

Comment:
Java is fine, I like this change.

Reply:
I scanned around github for things like this and it seemed like in the majority of cases there were no comments in between the rust use statements